RFPs/DRAFT-rfp-013-cryptonet-network-grant

```diff
@@ -0,0 +1,46 @@
+# RFP-013: Cryptonet Network Grant
```

Comment:
Are we calling them network grants or microgrants? The cryptonet website says the latter. I prefer the former, given these aren't microgrants (e.g. per FIL's definition, up to $5k).

@rosariogennaro ?

Reply:
Also note that the cryptonet page still mentions the exception for postdocs, which we agreed to remove. We should either update the website and sync back, or finalise this and push the changes to the website.